src/utils/tests/unist.test.mjs

```diff
@@ -0,0 +1,72 @@
+import { strictEqual } from 'node:assert';
+import { describe, it } from 'node:test';
+
+import { transformNodesToString, callIfBefore } from '../unist.mjs';
+
+describe('transformNodesToString', () => {
+  it('should transform a list of Nodes into a string', () => {
+    const nodes = [
+      {
+        type: 'paragraph',
+        children: [
+          {
+            type: 'text',
+            value: 'Hello, ',
+          },
+          {
+            type: 'strong',
+            children: [
+              {
+                type: 'text',
+                value: 'World',
+              },
+            ],
+          },
+        ],
+      },
+    ];
+    strictEqual(transformNodesToString(nodes), 'Hello, **World**');
```

Comment:
I wonder if we should replace this custom function with `remark.stringify` with `getRemarkRehype` in the future.

It would be interesting to know what is faster here, this method or `remark.stringify`

Reply:
How can you benchmark the code with the test runner? Personally, I've never seen anything like it.

Reply:
No not here, I mean somewhere else. nvm.